.env.example

```
@@ -1,16 +1,20 @@
-APNS_TEAM_ID=your_team_id_here
-APNS_BUNDLE_ID=in.juspay.shooter
-APNS_KEY="-----BEGIN PRIVATE KEY-----
-your_p8_key_content_here
------END PRIVATE KEY-----"
+# Authentication
+API_KEY=your-api-key-here
 
-# Device Token (from iOS app)
-DEVICE_TOKEN=your_device_token_from_ios_app
+# Apple Push Notification Service
+APNS_KEY=-----BEGIN PRIVATE KEY-----\nYOUR_KEY_HERE\n-----END PRIVATE KEY-----
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)